src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+    
+}
+
+.nav__item {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0;
+  font-size: 12px;
+  text-decoration: none;
+  cursor: pointer;
+  text-transform: uppercase;
+  color: #000;
+  line-height: 14px;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  width: 100%;
+  height: 4px;
+  left: 0;
+  bottom: 0;
+  background: #00acdc;
+  border-radius: 8px;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
```

Comment:
Group your selectors a little bit

```suggestion
.nav__link {
  display: block;
  padding: 23px 0;
  font-size: 12px;
  text-decoration: none;
  cursor: pointer;
  text-transform: uppercase;
  color: #000;
  line-height: 14px;
}

.nav__link:hover {
  color: #00acdc;
}

.is-active {
  position: relative;
  color: #00acdc;
}

.is-active::after {
  position: absolute;
  content: "";
  width: 100%;
  height: 4px;
  left: 0;
  bottom: 0;
  background: #00acdc;
  border-radius: 8px;
}
```

Reply:
Thank you for your help. I think I fixed everything. But during npm test it showed errors (pink waves on the page BackstopJS). How do I find out how to fix it? You see mine [TEST REPORT LINK](https://oksana-maruniak.github.io/layout_moyo-header/report/html_report/)?